troubleshooting/troubleshooting-android-whitelist-not-enabled.md

```diff
@@ -0,0 +1,62 @@
+---
+title: jQuery/AJAX/JSON Requests do not Work on Android
```

Comment:
I don't believe this title is descriptive and I don't like the slashes. Suggestion: 
Ajax jQuery Requests Do Not Work on Android